usr/share/rear/init/default/01_set_drlm_env.sh

```diff
@@ -6,8 +6,7 @@ fi
 
 PROGS=( "${PROGS[@]}" curl )
 
-drlm_import_runtime_config
+# Needed for curl with NSS support
+LIBS=( "${LIBS[@]}" /usr/lib64/libsoftokn3.so /usr/lib64/libsqlite3.so.0 /lib64/libfreeblpriv3.so )
```

Comment:
If you really must add those libraries manually, then please use a syntax like this:
```sh
        LIBS=( 
            "${LIBS[@]}"
            /usr/lib*/libsoftokn3.so* 
            /usr/lib*/libsqlite3.so* 
            /lib*/libfreeblpriv3.so*
        )
```

And please check if the libfreeblpriv3 is really always under `/lib*`.